mmv1/products/dlp/DeidentifyTemplate.yaml

```diff
@@ -42,8 +42,17 @@ examples:
     primary_resource_id: 'basic'
     test_env_vars:
       project: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'dlp_deidentify_template_with_template_id'
```

Comment:
Looking more closely at the behavior for this field, I would reduce the examples you've included:
- Both examples are very redundant with the existing `basic` examples, and I don't think they provide real value in the docs, so I would include `skip_docs: true` for both.
- For that same reason, it is probably not necessary to test all of the same fields, so ideally we could remove all but one of the `transformations`, and also remove optional fields like `description` and `display_name`.

Reply:
Sure. I've added `skip_docs: true` and removed redundant fields from both of these examples.